Automattic/vip-go-mu-plugins
# Search: Update ElasticPress submodule to latest from develop

## Description

Updates the ElasticPress submodule to the latest from the `develop` branch. This fixes an undefined variable notice (see https://github.com/10up/ElasticPress/issues/2155)

## Changelog Description

### Search: Update ElasticPress submodule

Updates ElasticPress to bring in a small fix for an undefined variable notice.

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Check out PR
2. Do regular search queries on a site
3. Update an index's settings:
4. `\ElasticPress\Elasticsearch::factory()->update_index_settings( 'vip-200508-post-1', array( 'index.number_of_replicas' => 2 ) );`
5. Above should not throw the undefined variable notice for `$closed`
